sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 # Release History
 
-## 3.0.0 (Unreleased)
+## 2.3.0 (Unreleased)
 
 ### Features Added
 
 - Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
     - Added a type `MicrosoftTeamsAppKind`.
     - Added a method `isMicrosoftTeamsAppIdentifier` to check if the identifier is `MicrosoftTeamsAppIdentifier`.
     - Added a field `teamsAppId` to `MicrosoftTeamsAppIdentifier`.
+    - With this version, any rawId starting with `28:` will map to MicrosoftTeamsAppIdentifier. Previously it mapped to `UnknownIdentifier`.
 
 ## 3.0.0-beta.2 (Unreleased)
```

Comment:
Let's remove this entry completeley

Reply:
We also need to change the package.json version to 2.3.0. Hopefully these two changes are enough to fix the failed CI Analyze step.

Reply:
Ok done